src/styles/main.scss

```
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Import each file separately with the specified extension

```suggestion
@import "./header.scss";
@import "./nav.scss";
@import "./catalog.scss";
```